core/css.ts

```diff
@@ -242,6 +242,10 @@ let content = `
   display: none;
 }
 
+.blocklyDisabled > .blocklyOutlinePath {
+  fill: darkgray !important;
+}
```

Comment:
I don't think you want to add this here. I think you want to modify the code in the zelos constants file `getCSS_`. You should be able to remove the `!important` declaration then.

And could you try just setting the stroke of the outline path instead of setting the fill? I think that might look nicer!